what4/what4.cabal

```diff
@@ -204,16 +205,36 @@ test-suite bvdomain_tests
   type: exitcode-stdio-1.0
   default-language: Haskell2010
 
-  hs-source-dirs: test
+  hs-source-dirs: test, test/QC
   main-is: BVDomTests.hs
 
-  other-modules:
+  other-modules:  VerifyBindings
 
   build-depends: base
                , parameterized-utils
                , tasty >= 0.10
                , tasty-quickcheck >= 0.10
                , QuickCheck
+               , transformers
+               , what4
+
+  ghc-options: -Wall -Wcompat -Werror=incomplete-patterns -Werror=missing-methods -Werror=overlapping-patterns
+
+test-suite bvdomain_tests_hh
```

Comment:
Having two `test-suite` stanzas will run the test both ways right? Maybe we should use a cabal flag to choose between them instead.

Reply:
It generates two executables: one that uses QuickCheck and one that uses Hedgehog.  A cabal flag could choose but then we'd only build one of those: this way I'm actually verifying that the QuickCheck and Hedgehog mapping both work.